backend/api.py

```diff
@@ -13,23 +13,32 @@
 logging.basicConfig(level=logging.INFO)
 
 class HintRequest(BaseModel):
-    name: str = Field(..., example="Two Sum")
```

Comment:
Curious why we don't want to pass in the question name anymore?

Reply:
We don't have the code to scrape it, it was never being passed. We will pass the name along with other potentially useful data like the problems 'Related Topics' in #18